### ISSUE TITLE
no-require-imports complains on alias import

### ISSUE THREAD

**Issue by antonmoiseev** [palantir/tslint#585]
Valid alias import causes the error, though it doesn't have a `require()` on the RHS:

``` ts
import Service = MyApp.Service; // (no-require-imports): require() style import is forbidden
```

**Comment by andy-hanson** [comment]
Looks like this is already fixed in latest.
